flocker/control/test/test_httpapi.py

```diff
@@ -1309,6 +1309,143 @@ def test_container_with_volume(self):
 )
 
 
+class UpdateContainerConfigurationTestsMixin(APITestsMixin):
+    """
+    Tests for the container configuration update endpoint at
+    ``/containers/<containername>``.
+    """
+    def _create_container(self):
+        """
+        Utility function to create a container configuration via the API.
+        """
+        saving = self.persistence_service.save(Deployment(
+            nodes={
+                Node(hostname=self.NODE_A),
```

Comment:
Maybe these should each have unrelated application on them, so that you can see that it's actually moving and not wiping out existing unrelated applications.